src/pages/base.ts

```diff
@@ -24,11 +24,11 @@ class Page {
     const url =
       data.raw().reduce((prev, item) => {
         if (prev.indexOf(`:${item[0]}`) === -1) {
-          additionalParams.push(item);
+          additionalParams[item[0]] = item[1];
```

Comment:
What is a reason for this change?

Reply:
It was changed in @tgorskievo PR https://github.com/TheSoftwareHouse/Kakunin/pull/211/files

Reply:
Ok I know why

Reply:
Did you create this branch from a current `next`? 

I updated merged `master` to `next` in a separated PR which includes missing doc version `3.0.1`